oggm/utils/_workflow.py

```diff
@@ -203,12 +188,42 @@ def show_versions(logger=None):
         except BaseException:
             deps_blob.append((modname, None))
 
+    return deps_blob
+
+
+def get_git_ident():
+    ident_str = '$Id: f2cb553755951450761f5ee64e4a4c433ef61521 $'
```

Comment:
This is not going to work unless you also change .gitattributes accordingly.

Reply:
Thanks - will do